ext/pgsql/pgsql.c

```diff
@@ -5865,20 +5920,26 @@ PHP_FUNCTION(pg_select)
 	long result_type = PGSQL_ASSOC;
 	PGconn *pg_link;
 	zend_string *sql = NULL;
-	int argc = ZEND_NUM_ARGS();
 
-	// TODO: result_type is unused by zpp!
-	if (zend_parse_parameters(argc, "rsa|l",
+	/* TODO Document result_type param on php.net (apparently it was added in PHP 7.1) */
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "rsa|ll",
```

Comment:
If you add it here, it also needs to be added in the stub. And I think there needs to be a test for it, given how it was "added" last time in PHP 7.1...

Reply:
Adding a case using PGSQL_NUM to ext/pgsql/tests/pg_select_001.phpt should do it for a test.